### PR TITLE
PR breaking change release note change

### PR DESCRIPTION
**Pull request by david-yu**
Changes proposed in this PR:
- Update changelog for #2392 

How I've tested this PR:

How I expect reviewers to test this PR:

Checklist:
- [ ] Tests added
- [ ] CHANGELOG entry added 
  > HashiCorp engineers only, community PRs should not add a changelog entry.
  > Entries should use present tense (e.g. Add support for...)

### CODE REVIEW

**Review by lkysow — COMMENTED** [review]
You can add two release notes in the same file so I'd recommend doing both a bug and breaking-change.

File: .changelog/2392.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-```release-note:bug
-control-plane: Always update ACL policies upon upgrade
-```
\ No newline at end of file
+```release-note:breaking-change
+control-plane: Always update ACL policies for consul-connect-injector upon upgrade.
```
Comment (lkysow) [inline .changelog/2392.txt]: The previous bug changelong can remain the same (we're updating all policies not just injector)

The breaking change would be: "All policies managed by consul-k8s will now be updated on upgrade. If you previously edited the policies after install, your changes will be overwritten."

Comment (lkysow) [reply .changelog/2392.txt]:
```
    ```release-note:bug
    control-plane: Always update ACL policies upon upgrade.
    ```
    
    ```release-note:breaking-change
    control-plane: All policies managed by consul-k8s will now be updated on upgrade. If you previously edited the policies after install, your changes will be overwritten.
    ```
```

**Review by lkysow — APPROVED** [review]

File: .changelog/2392.txt (hunk 1 above)
Comment (david-yu) [reply .changelog/2392.txt]: Ok thanks! That was helpful!

**Pull request merged**